src/Main/RestaurantsContainer/RestaurantsContainer.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react'
+import './RestaurantsContainer.scss'
+
+const RestaurantsContainer = () => {
+
+    const [restaurants, setRestaurants] = useState([])
+
+    useEffect(() => {
+        fetch('http://localhost:8080/restaurants')
+            .then(response => {
+                return (response.json())
+            }).then((data) => {
+                setRestaurants(data)
```

Comment:
No error handling - mentioned in API readme, but not sure if necessary?

Reply:
Resolved